jobs/anac-certified-attributes-importer/package.json

```diff
@@ -3,7 +3,7 @@
   "main": "./dist/index.js",
   "type": "module",
   "scripts": {
-    "test": "vitest run",
+    "test": "vitest --no-threads run",
```

Comment:
for some reason the new tests raises a segmentation fault.
I disabled threads following [this issue](https://github.com/vitest-dev/vitest/issues/317)